reference/config_files/editable_layout.rst

```diff
@@ -0,0 +1,40 @@
+.. _editable_layout:
+
+Editable layout files
+=====================
+
+This file contain information consumed by :ref:`editable packages <editable_packages>`. It is
+an *.ini* file listing the directories that Conan should use for the packages that are opened
+in editable mode:
+
+.. code-block:: ini
+
+    # Affects to all packages but cool/version@user/dev
```

Comment:
Something like `# Affects to all packages but cool/version@user/dev (this reference has its own dedicated section)`?

Or maybe it is better to divide this into two code-blocks:
 * First one with sections without references, just showing relative paths, absolute ones and the placeholders things.
 * Second code-block with references and without to show and explain the behavior